sys/net/gnrc/routing/rpl/gnrc_rpl_internal/validation.h

```diff
@@ -46,8 +43,6 @@ static inline bool gnrc_rpl_validation_DIS(gnrc_rpl_dis_t *dis, uint16_t len)
         return true;
     }
 
-    DEBUG("RPL: wrong DIS len: %d, expected: %d\n", len, expected_len);
```

Comment:
I would suggest to add a DEBUG message where these functions are used (its only in one file), i.e. in `gnrc_rpl_control_messages.c`

Reply:
As an alternative solution, I could also move these functions as to `gnrc_rpl_control_messages.c` and keep the `DEBUG` messages in.

Reply:
I agree DEBUG output shouldn't be in header files, but also we need debug output - hence my point: don't remove but move them. I don't have a preference how, so you can proceed with your idea.

Reply:
Thanks for the reply!